src/Files.App/Views/Layouts/BaseLayoutPage.cs

```diff
@@ -1200,6 +1216,10 @@ private void RefreshItem(SelectorItem container, object item, bool inRecycleQueu
 
 		protected internal void FileListItem_PointerPressed(object sender, PointerRoutedEventArgs e)
 		{
+			// Set can window to front and bring the window to the front if necessary
```

Comment:
Please add issue number like this.
There's also 5 places to add this below this line.
```suggestion
			// Set can window to front and bring the window to the front if necessary (#13255)
```